odoo/tools/translate.py

```diff
@@ -600,6 +600,8 @@ def __iter__(self):
             comments = "\n".join([c for c in entry.comment.split('\n') if not c.startswith('module:')])
             source = entry.msgid
             translation = entry.msgstr
+            # if any of the `code` occurrence is a webclient path, flag entry as web
+            web = any(re.match(r'code:.*/static/src/(js|xml)', occurrence) for occurrence, _ in entry.occurrences)
```

Comment:
No, because the web flag is based on checking the content of all occurrences, whereas the `code` handler will skip everything after its own first match. So if the term e.g. has occurrences in python then occurrences in JS, the code handler inside the loop will only trigger for the first one.